plugins/org.eclipse.osee.define/src/org/eclipse/osee/define/operations/publisher/publishing/GeneralPublishingWordTemplateProcessorServer.java

```diff
@@ -71,98 +71,105 @@ public GeneralPublishingWordTemplateProcessorServer(OrcsApi orcsApi, AtsApi atsA
-                */
-
-               this.overrideClassification,
-
-               /*
-                * When recursing, this tester accepts or rejects descendant artifacts
-                */
-
-               ArtifactAcceptor.and
-                  (
-                     this.wordRenderApplicabilityChecker,
-                     this.emptyFoldersArtifactAcceptor
-                  ),
-
-               /*
-                * Client/Server calling of the Data Rights Manager is different.
-                */
-
-               this.dataRightsOperations::getDataRights
-
-            )
-         .ifPresent
-            (
-               ( datarightsContentBuilder ) ->
-                  artifacts
-                     .forEach
-                        (
-                           ( artifact ) -> this.processArtifact
-                                              (
-                                                 artifact,
-                                                 wordMl,
-                                                 ArtifactAcceptor.ok(),
-                                                 datarightsContentBuilder,
-                                                 PublishingArtifactLoader.CacheReadMode.LOAD_FROM_DATABASE,
-                                                 IncludeBookmark.YES.getArtifactAcceptor(),
-                                                 null   /* Artifact Post Processor */
-                                              )
-                        )
-            );
+      if (artifacts.isEmpty()) {
+
+         wordMl.addParagraphNoEscape("No artifact(s) have been published. Make sure that your artifact(s) are applicable for the view.");
+
+      } else {
+
+         this.emptyFoldersArtifactAcceptor =
+            WordRenderUtil.populateEmptyHeaders
+               (
+                  artifacts,
+                  this.includeHeadings,
+                  this.allowedOutlineTypes,
+                  this.headingArtifactTypeToken,
+                  this.contentAttributeType,
+                  this.excludedArtifactTypeArtifactAcceptor,
+                  ArtifactAcceptor.ok()
+               );
+
+         /**
+          * Setup Data Rights for the publish.
+          * <p>
+          * DataRightsClassification override comes in as a publishing option string, compare string to all
+          * DataRightsClassifications, if they match, set override variable to that classification. This override makes it
+          * that the entire published document uses the same data rights footer, regardless of the attribute on artifacts.
+          * <p>
+          * Given the list of artifacts for the publish, this loops through and adds any recursive artifacts to also be
+          * published (if specified through recurseChildren) and determines all of their data rights.
+          */
+
+         //@formatter:off
+         WordRenderUtil
+            .getDataRights
+               (
+                  /*
```

Comment:
If we're going to have this much javadoc for the parameters, it should be on the method declaration as @param rather than each individual param when calling it.

Reply:
They are also defined as @params in the .getDataRights javadoc... but I am hesitant to remove any javadoc that I didn't write just yet